pkg/plugins/policies/matchers/match.go

```diff
@@ -55,6 +55,9 @@ func MatchedPolicies(rType core_model.ResourceType, dpp *core_mesh.DataplaneReso
 		ToRules: core_xds.ToRules{
 			Rules: toRules(dpPolicies),
 		},
+		SingleItemRules: core_xds.SingleItemRules{
```

Comment:
Isn't that just `DataplanePolicies`?

Reply:
`DataplanePolicies` is all policies, single item ones are filtered [here](https://github.com/kumahq/kuma/blob/cf879a815bafed57dff7696f7cb5094300906b2c/pkg/plugins/policies/matchers/match.go#L99). TBH I just asked @lobkovilya how he'd implement this and this is what we ended up agreeing on.

Reply:
So what's the us of `DataplanePolicies` then?

Reply:
I'm guessing the same question applies then to `FromRules` and `ToRules` (and it's party directed to @lobkovilya):

1. I thought that bucketing these (to / from / single) is resource heavy so we want to have them pre-computed.
2. If nothing accesses `DataplanePolicies` (regardless of their type) maybe we can drop that field.

Reply:
I think you are right both times. But @lobkovilya can confirm

Reply:
So can I leave this field as it is until someone implements inspect API?

Reply:
Yes, it's not the scope of this PR

Reply:
Not sure why Ilya's comment is not appearing here but it's a bit lower. I'm closing this to be resolved when inspect API PR raised.